### DISCUSSION TITLE
Default filament table pagination display is not working perfectly

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#9281] [General]
### Description

In my project i use filament for dashboard and i use default table but in my table footer paigination is not show default style .Please give me advice or how to solve to defalut styel. why this happen?
![Screenshot (128)](https://github.com/filamentphp/filament/assets/76741291/48bbe8e7-4d10-4675-816e-d12d4f4d3f7f)

**Comment by VincvanAs** [comment]
I had the same issue, running npm update, npm run build and clearing cache solved it for me

**Reply by ayemin-tun** [reply]
Thank! it solve me as well when i make filament upgrade

**Comment by R70YNS** [comment]
I had the same issue but hadn't included the necessary CSS i.e. `@filamentStyles` in `app.blade.php` 

Making sure I had all dependencies in app.blade.php and then recompiling solved the issue for me.
